tests/nnet/test_conv.py

```diff
@@ -240,7 +240,7 @@ def test_conv_1d_bkwd():
     behavior with `constant` arg, good behavior of null_gradients, etc."""
 
 
-@settings(deadline=None)
+@settings(deadline=None, suppress_health_check=(HealthCheck.filter_too_much,))
```

Comment:
can you craft the input such that this is not necessary?

Reply:
So, I was thinking about that, and I think that it is actually a relatively painful task. I am satisfied with this for now - this usually actually passes the health check, it was just on occasion that it was failing